agenthub/codeact_agent/codeact_agent.py

```diff
@@ -265,3 +263,12 @@ def step(self, state: State) -> Action:
 
     def search_memory(self, query: str) -> List[str]:
         raise NotImplementedError('Implement this abstract method')
+
+    def logCost(self, response):
+        cur_cost = self.llm.completion_cost(response)
+        self.cost_accumulator += cur_cost
+        logger.info(
+            'Cost: %.2f USD | Accumulated Cost: %.2f USD',
+            cur_cost,
+            self.cost_accumulator,
+        )
```

Comment:
not snake_case. Also, why are you moving this to a function?

Reply:
Thanks updating naming convention.  Using a function here is a cleaner implementation, you don't need to clutter the main function with 6/7 lines of simple logging code